### DISCUSSION TITLE
Confused about VBA Stomping Message - how should I proceed?

### DISCUSSION THREAD

**Discussion by mtxnk** [decalage2/oletools#777] [Q&A]
**Apologies - I am not an expert in Malware.** 
After a ppam I wrote has been flagged by an antivir VIRUSBARRIER on Mac I used your great tools to analyze.
One attempt was to delete all code Modules and analyze the nuked file.
I get the info

```
|Suspicious|VBA Stomping        |VBA Stomping was detected: the VBA source    |
|          |                    |code and P-code are different, this may have |
|          |                    |been used to hide malicious code
```

It seems there is a Form code left which was not removed by the removal code 
(I used a PPT adapted version of [this code by Jacob Hilderbrand at VBA EXPRESS](https://www.vbaexpress.com/kb/getarticle.php?kb_id=93)

I see a lot of code in the form debug like this:
```
DEBUG    Printable string found in form: b'd\xd8\xc5\x86\xd6\xca\xe3\xff\xff\xf6\xff\xff\x96\xe6\xdat\xd1\xc3\xa6\xe4\xdb\xb4\xf5\xec\xdb\xff\xfe\xbe\xd0\xd3\xb4\xec\xe4^\xdb\xc7\xfd\xff\xff(LO'
DEBUG    Printable string found in form: b'>@\xee\xff\xff\xd3\xeb\xee\xd4\xff\xfc\x90\xa9\xaa\xa6\xed\xe3'
```

I read that stomping detection is experimental. How should I proceed? Any advice?

THANKS!

**Comment by decalage2** [comment]
In practice VBA stomping detection triggers a lot of false positives, so in your case I would not worry about it. This is only useful when analyzing actual malware, which does not seem to be your case as you wrote the macro.

It's not really clear to me what you are trying to do: why not simply delete macros and forms using the VBA editor?
